docs/source/config.json

```diff
@@ -100,6 +100,7 @@
     "Telemetry and Monitoring": {
       "Overview": "/configuration/telemetry/overview",
       "GraphOS reporting": "/configuration/telemetry/apollo-telemetry",
+      "Client awareness": "/managed-federation/client-awareness",
```

Comment:
This page gets lost in the current nav structure. (Maybe that was the intention since it's part of another docset?)

Since it's part of the top-level "Telemetry and Monitoring" section, it may make more sense here.